### ISSUE TITLE
App support 

### ISSUE THREAD

**Issue by Luca-Macari** [AntonyCorbett/OnlyT#67]
Hi there, 

I would like to understand how to use OnlyT with a remote application from an iOS device. Could you explain it in layman’s terms please. 

Kind regards 
Luca Macari

**Comment by AntonyCorbett** [comment]
Hi Luca

You can't _control_ the timer from a remote app yet as there are no compatible apps available. One is currently being written.

However, you can _display_ the timer on a remote device. Please see the Wiki for more information.

Kind regards
Antony

**Comment by Luca-Macari** [comment]
Hi Antony

Thank you for such a quick response. Ok I will wait patiently :)
Keep up the good work

Regards
Luca

**Comment by MattDelmarter** [comment]
Hi Luca,

I have released a mobile app for iOS that can be used to remotely control OnlyT. Please see here:
https://onlyt.app

Let me know how it works for you.

Matthew
